core/space/services/services_keypair.go

```diff
@@ -5,38 +5,28 @@ import (
 	"encoding/hex"
 
 	"github.com/FleekHQ/space-daemon/core/keychain"
-	"github.com/FleekHQ/space-daemon/core/space/domain"
 	"github.com/FleekHQ/space-daemon/core/textile/hub"
 )
 
-func (s *Space) GenerateKeyPair(ctx context.Context, useForce bool) (domain.KeyPair, error) {
+func (s *Space) GenerateKeyPair(ctx context.Context, useForce bool) (string, error) {
```

Comment:
Added a comment in 4944baf
I think the new interface is better, since the consumer can choose what to do with the mnemonic, and the key pair didn't offer any use currently, since we might want to avoid exposing the private key and the public key can be fetched with `GetPublicKey`